src/plugins/stake-plugins/generic/util/EdgeWalletSigner.ts

```diff
@@ -74,13 +74,17 @@ export class EdgeWalletSigner extends ethers.Signer {
     }
 
     const data = await transaction.data
+
     const memoHexValue = data == null ? undefined : typeof data === 'string' ? data.replace('0x', '') : base16.stringify(data)
     const memos: EdgeMemo[] = memoHexValue == null ? [] : [{ type: 'hex', value: memoHexValue }]
+
+    const customData = await transaction.customData
```

Comment:
This is weird. I would have expected `transaction.customDataPromise` as a clearer name, but I guess it's out of scope for this PR.

Reply:
This whole first commit doesn't seem to be used, in general. Since we are super-late in staging, please move it to a new "Staking cleanups" PR on develop, so we don't take any extra risk. Or, if it's needed for some reason, please explain.

Reply:
The type is "Promise<T> | T ...", so I have to resolve it to remove the promise type